components/toast/index.css

```diff
@@ -59,6 +59,7 @@ governing permissions and limitations under the License.
   display: flex;
   flex: 0 0 auto;
   align-items: flex-start;
+  border-inline-start-color: rgba(255, 255, 255, 0.2);
```

Comment:
We put colors in the skin.css. Can you have border-color still in skin.css and then set the border-width for inline-start in the index.css file?

Reply:
Also, make your prs against the `spectrum-tokens-17` branch while we're working on it make it easier to merge.

Reply:
This commit moved the color out of the skin file. The reason I did so was to fix the bug. When compiled for color stops when the css is in `skin.css`, the css compiles like so:
````
.spectrum--light [dir="ltr"] .spectrum-Toast-buttons {
  border-left-color: rgba(255, 255, 255, 0.2);
}
````
That does not work because it makes assumptions on the hierarchy where you need to have nesting like this: `.spectrum--light` > `[dir="ltr"]` > `.spectrum-Toast-buttons`

Changing the order of processing can turn that into:
````
[dir="ltr"] .spectrum--light .spectrum-Toast-buttons {
  border-left-color: rgba(255, 255, 255, 0.2);
}
````
But that assumes a hierarchy of `[dir="ltr"]` > `.spectrum--light` > `.spectrum-Toast-buttons`

My current implementation applies `.spectrum--light` and `[dir="ltr"]` both to the same lavel i.e. body so neither of those assumed hierarchies are true. I can obviously change my implementation, but I guess that begs the question of whether you want that restriction.

Keeping the color (which is the same for all color stops) in the `skin.css` file also results in the css being duplicated four times for color stops.